src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/InputEmptyCase.java

```diff
@@ -0,0 +1,36 @@
+////////////////////////////////////////////////////////////////////////////////
+// Input test file for testing empty LITERAL_CASE.
+// Created: 2017
+////////////////////////////////////////////////////////////////////////////////
+package com.puppycrawl.tools.checkstyle.checks.blocks;
+
+class InputEmptyCase
+{
+    void method1(int a) {
+        switch (a) {}
+        switch (a) {case 1: ; } // no violation
+        switch (a) {case 1:{}}  // 1 violation
+        switch (a) {
+            case 1:
+        }
+        switch (a) {
+            case 1: // 1 violation
+            {}
+        }
+        switch (a) {
+            case 1: // 1 violation if checking statements
+            {// none if checking text
+            }
+        }
+    }
+
+    public void method2(char c) {
+        switch(c) { case 0: } // no violation
+        switch(c) { case 0: {} } // 1 violation
+        switch(c) { case 0: method1(0); {} } // no violation
+        switch(c) { case 0: case 1: {} } // 1 violation
+        switch(c) { case 0: {} case 1: {// 2 violations if checking statements
+        }
+        }
+    }
+}
```

Comment:
it is good that you think about such cases but they are not required to be covered.
CASE by definition without "{ ...}" is working in block of SWITCH , so share the same scope as SWITCH.
This Check care about block that goes RIGHT AFTER defined token. 

This check should validate only first `{ }` in your case and raise violation. Please make update for xdoc to mention this to user. We will ignore super-weird code.

This Check does not validate sequential blocks `{}`,  you make put blocks at any possible place in code.

@rnveach , please confirm expected behavior.